compose.static.yml

```
@@ -0,0 +1,141 @@
+services:
+
+  bootstrap:
+    build:
+      context: .
+      dockerfile: testnet/Dockerfile
+    ports:
+      - "3000:3000/tcp"
+      - "18080:8080/tcp"
+    volumes:
+      - ./testnet:/etc/nomos
+    depends_on:
+      - mix-node-0
+      - mix-node-1
+      - mix-node-2
+    environment:
+      - BOOTSTRAP_NODE_KEY=${DOCKER_COMPOSE_BOOSTRAP_NET_NODE_KEY:-1000000000000000000000000000000000000000000000000000000000000000}
+      - LIBP2P_NODE_MASK=${DOCKER_COMPOSE_LIBP2P_NODE_KEY_MASK:-2000000000000000000000000000000000000000000000000000000000000000}
+      - LIBP2P_REPLICAS=3
+      - OVERLAY_NODES=${DOCKER_COMPOSE_OVERLAY_NODES:-1000000000000000000000000000000000000000000000000000000000000000}
+    entrypoint: /etc/nomos/scripts/run_bootstrap_node.sh
+
+  libp2p-node-1:
+    build:
+      context: .
+      dockerfile: testnet/Dockerfile
+    volumes:
+      - ./testnet:/etc/nomos
+    depends_on:
+      - bootstrap
+      - etcd
+      - mix-node-0
+      - mix-node-1
+      - mix-node-2
+    ports:
+      - "3001:3000/tcp"
+      - "18081:8080/tcp"
+    environment:
+      - LIBP2P_REPLICAS=3
+      - ETCDCTL_ENDPOINTS=${DOCKER_COMPOSE_ETCDCTL_ENDPOINTS:-etcd:2379}
+      - ETCDCTL_API=${DOCKER_COMPOSE_ETCDCTL_API:-3}
+      - LIBP2P_NODE_MASK=${DOCKER_COMPOSE_LIBP2P_NODE_KEY_MASK:-2000000000000000000000000000000000000000000000000000000000000000}
+      - OVERLAY_NODES=${DOCKER_COMPOSE_OVERLAY_NODES:-1000000000000000000000000000000000000000000000000000000000000000}
+      - OVERLAY_SUPER_MAJORITY_THRESHOLD=${DOCKER_COMPOSE_SUPER_MAJORITY_THRESHOLD:-1}
+      - NET_INITIAL_PEERS=${DOCKER_COMPOSE_NET_INITIAL_PEERS:-/dns/bootstrap/tcp/3000}
+    entrypoint: /etc/nomos/scripts/run_nomos_node.sh
+
+  libp2p-node-2:
+    build:
+      context: .
+      dockerfile: testnet/Dockerfile
+    volumes:
+      - ./testnet:/etc/nomos
+    depends_on:
+      - bootstrap
+      - etcd
+      - mix-node-0
+      - mix-node-1
+      - mix-node-2
+    ports:
+      - "3002:3000/tcp"
+      - "18082:8080/tcp"
+    environment:
+      - LIBP2P_REPLICAS=3
+      - ETCDCTL_ENDPOINTS=${DOCKER_COMPOSE_ETCDCTL_ENDPOINTS:-etcd:2379}
+      - ETCDCTL_API=${DOCKER_COMPOSE_ETCDCTL_API:-3}
+      - LIBP2P_NODE_MASK=${DOCKER_COMPOSE_LIBP2P_NODE_KEY_MASK:-2000000000000000000000000000000000000000000000000000000000000000}
+      - OVERLAY_NODES=${DOCKER_COMPOSE_OVERLAY_NODES:-1000000000000000000000000000000000000000000000000000000000000000}
+      - OVERLAY_SUPER_MAJORITY_THRESHOLD=${DOCKER_COMPOSE_SUPER_MAJORITY_THRESHOLD:-1}
+      - NET_INITIAL_PEERS=${DOCKER_COMPOSE_NET_INITIAL_PEERS:-/dns/bootstrap/tcp/3000}
+    entrypoint: /etc/nomos/scripts/run_nomos_node.sh
+
+  libp2p-node-3:
+    build:
+      context: .
+      dockerfile: testnet/Dockerfile
```

Why not just specify the path only?
```suggestion
    build: 'testnet'
```